### PR TITLE
Fixed proxy types matching

### PR DESCRIPTION
**Pull request by antonijzelinskij**
#8693e7xgf

### CODE REVIEW

File: feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/signer/proxy/callFilter/CompoundCallFilter.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import jp.co.soramitsu.fearless_utils.runtime.definitions.types.generics.GenericCall
+
+class CompoundCallFilter(
+    private val filters: List<CallFilter>
+) : CallFilter {
+
+    constructor(vararg filters: CallFilter) : this(filters.toList())
+
+    override fun canExecute(call: GenericCall.Instance): Boolean {
+        return filters.all { it.canExecute(call) }
```
Comment (valentunn) [inline feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/signer/proxy/callFilter/CompoundCallFilter.kt:12]: I think it should be `any`

Comment (antonijzelinskij) [reply feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/signer/proxy/callFilter/CompoundCallFilter.kt:12]: Of course, thank you!

Comment (ERussel) [inline feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/signer/proxy/callFilter/CompoundCallFilter.kt:12]: As I understand we need to match only one of the filter - not all of them.

**Comment by stepanLav** [comment]
Task linked: [CU-8693e7xgf Android. Not enough permissions for start staking, vote on Staking, Governance proxy](https://app.clickup.com/t/8693e7xgf)

**Review by ERussel — APPROVED** [review]

**Pull request merged**